data/json/recipes/recipe_others.json

```diff
@@ -2110,6 +2110,28 @@
   ]
 },{
   "type" : "recipe",
+  "result": "holy_symbol_wood",
+  "category": "CC_OTHER",
+  "subcategory": "CSC_OTHER_OTHER",
+  "difficulty": 0,
```

Comment:
This recipe could probably use fabrication skill. Carving stuff out of wood with a knife is not something that most people do everyday, so you're supposed to learn something from doing it.